packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -60,6 +60,11 @@ export interface DatepickerCalendarProps extends UIComponentProps, Partial<ICale
   /** A render function to customize how the calendar grid row is rendered. */
   calendarGridRow?: ShorthandValue<DatepickerCalendarGridRowProps>;
 
+  /**
+   * The currently selected date range.
```

Comment:
```suggestion
   * The currently selected date range, currently only supports week
```

Reply:
We could also retype `dateRangeType` explicitly in the props to be `DateRangeType.Week` if that's all we're using it for